docs/metrics/ga.md

```diff
@@ -185,7 +203,8 @@ Here is a list of dimensions we are currently using
 | Home Page, Experiment Detail Page, Share Page | Which test has this user been selected for?       | 8         | {'installButtonBorder', ''} |
 | Home Page, Experiment Detail Page, Share Page | Which cohort has this user been selected for?     | 9         | {'bigButton', ''} |
 | All                                           | In which responsive breakpoint is the user?       | 10        | {'mobile','small','medium','big'} |
-| All experiment-specific locations and events  | On which experiment did this action take?         | 11        | experiment slug | 
+| All experiment-specific locations and events  | On which experiment did this action take?         | 11        | experiment slug |
+| Featured Experiment Video Progress            | Duration of Featured YouTube video                | 12        | {n}
```

Comment:
In general, custom dimensions should be for categorical variables. Even the numeric-based custom dimensions above (e.g. cd3) have such a narrow range of possible values that they are effectively categorical. Depending on the questions you want this to answer, perhaps we could make it categorical by grouping users like this:

- Video not watched (<= 10% of duration)
- Video fully watched (>=90% of duration)
- Video partially watched (all others)